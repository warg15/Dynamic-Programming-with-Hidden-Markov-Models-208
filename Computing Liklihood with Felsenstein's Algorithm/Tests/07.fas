>Common-cuckoo
CGTATGTAGAGGTCGATCTTCACATGTGAAGCAATGTACGTAGTGGTTAACAGTTCTGGGAAGCCATGAGTACTAATATGGGCCCTATTGAACGCCACCGGAAGATGCGAGTTCGGCTAACTTCGCGTGAATGTTGATTAACGTAGCGATTTCTAGTAAAGCATGGTGCAACATAAGTGACACTTTCTGACGGAGAGGTT
>Pekin-duck
CTTATGAAGAGGTCGATCTCCACGTGTGCAGTGATGTAAGTAGTAGCTAACAGCTCTGAGAAACCATGAGTACTAAGACGGGCTCAATTGAATGTCAACGGCTAAGATCATTGCGACTGACTTCGCATGAATGTTGATTGAGGTACTGATTTATAGCAAAGCATGATGCAACATGAGTGATCTTTTGTGAAGGGGAGATT
>Red-crested-turaco
CCTATGTAGAGGTTGACCTGCACGTGTGTGGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAAGTTGGGCTCAATAGAACGTCAACGGCTGATTTGAGTCCGGCTTACTTCGCATGAATGTTGATTAATATACCGATTTCTAGCAAAACATGGTGCAACATGAGTGACACTTTCTGAAGGGGAGGTT
>Great-crested-grebe
CTTATGTAGAGGTTGATCTGCACGTGAGAAGTAATGTAAGTAGTGGCTAGGAATTCTGGGAAGCCATGAGTACTAAGAAGGGCTCAAGTAAACGTCGGCGGCGGATGCGAGATCGGCTGACTTCGCATTAATGTTGATTAATGTACCGATCTATAGCATGGCATGGTGCAACATGAGCGACACTTTCTGAAGGGGTAGTT
>Red-legged-seriema
CTTATGTAGAAGTTGATCCGCACGTGTGAAGTAATCTAAGTATTGGCTAACAATTCTGGGAGGCCATGAGTACTAAGATAGGCTCAATTGAACGTCAACGGCTGATGCGAGTTCGGCTGACTTCGCATGAATGTTGATTAATGTACCTATTTCTAGCAAAGCATGGTACAACATGAGTGACGCTTTCTGAAGGGGAGGTT
>Chicken
GCTATAAGAGGGTTGATCTCCACGTGTAAAGCGATATAAGTAATGGTTAACAGCTTGGGAAAGCCATGAATACTAAGATGGGTTTAATTCAATGTCAACGGCTAAGACCATTGCGGTTGGCTTCGCATGAATGTTGCCTAATGTACCGATTTCTGGCAAAGTATGATGCAACATGAGTGATTTTTTCTAAGGGAGAAATT
>Barn-owl
CCCTTGTAGAAGTTGATCCGCACGTGTCAAGTAATCTAAGTAGTGGCTAACAGTTCTGGGAGGCCATGAGTACTAAGATGGGCTCAATTGAACGTCAACGGCTGATACGCGTTCGGCTGACTTCGCATAAATGTTGATTAATGTACCGATTAATAGCAAAGTCTGGTACAACATGAGTGACACTTTCTGAAGGGGAGGTT
>Adelie-penguin
CTTATGTAGAGGTTGATCCGCACGTGTGAAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAGAATGTGCTCAATTGAACATCAACGCACGATGCGAGTTCGGCTGACTTCGCATGAATGTTGCCTAATGTACCGATTTCTAGCAAAGCATGGTGCAAAATGAGTGACACTTTCTGAAGGGGAGGTT
>MacQueens-bustard
CTTATGTAAAGGTTGATATGCACGCGTGAAGTAATGTAAGTAGGGGCTAACAGTTCTGGGAAGCCATGAGTACTGAAATCGGCTCAATTGAACGTCAACTGCTGATCCGAATTCGGCTGACCTCGCATGAATGTTAATTAATGTACCGATTTCTGACAAAGCATGCTGCAACATGAGTGACACTTCGTGAAGGGGAGGTT
>Chuck-wills-widow
CTTATGTAGAGGTTGATCTGCACGTGTGAAGTAGTGTAAGTAGTGGCTAACAGTCCTGGGAAGCCATGCGTACTAAGATAGGCTTAATTGAACGTCAACGGCTTATGCGAGTTTTGCTGACTTCGCATGAATGTTGATTAATGTACCGACTTCGAGCAAAGCATAGTGCAATATGAGTGACAGTTACTGAGGGGGAGGCC
>Red-throated-loon
TTTATCTAGAGGTTGATCTGCACGTGTGAAGCAATGTAAGTAATGGCTAATAGTTCTGGGAAGCCATGAGTACTAAGATGGGCTCAATTGAACGTCAACGGCTGATGCGAGTTCGGCTGACCTCGCATGAATGTTGATTAATGTACCGATTTCTAGTGAAGCATGGTGCAACATGAGTGACACTTTCTGAAGGGGAGGTT
>Killdeer
TTTATGTAGAGGTCGATCTGCTCGTGTGAAGTAATGTAAATAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAAGATGGGCTCAACTGAACGTGAACGGCTGATGCGAGTTCGGCAGACTTCGCACGAATGTTGATTAGTGTACCGATTTCTAGCAAAGCATCGTCTAACATGAGCGACACTTTCTAAAGGGGAGGTT
>Downy-woodpecker
CTCATGTGGAAGTTGATCTGCACTCATAAAGTAATCTAGGTAGTGGCTAACAGATTTGAGAGGGCATCAGTACTAAGATGGCCTGAATTGATCGTTAACGGCTGATGCGAGTTCGACTGAGTTCGCATGAATGTCTATTAATGTACCGATTTCTAGAACAGCACGGTACAATATGAGTGACACTTTCTGAATTGGAGGTT
>Turkey
GTTATAAGGAGGGTGATCTCCACGTGTAAAGCGATGTAAGTAATGATTAACAGCTCGCGAAAGCCATGAATACTAAGATGGGTTTAATTGAATGTCAACGGCTAAGACCACTGCGGTTGGTTTCGCATGAATGTTGCCTAATGTACCGATTTCTGGCAAAGCATGGTGCTATATGAGCGATCTTTTCTGAGGGGGAGATT
>Hoatzin
CTTATGTGGAGGTTGATCTGCACATGTGAAGTAATGTAAATAGTGGCTAACAGTTCTGGGAAGCCATGAACACTAAGGTGAGCTCAGTTGAACGTCAACGGCTGATGGGAGTTCGGCTGACTTCGCATGAATGTTGATTAATACACCGATTTCTAGCGAAGCATGGTGCAACATGAGTGGCACTTTCTGCTGGGGCGGTT
>Sunbittern
CTTACGCAGAGGTTGATCTGCACGTGTGAAGTAATGTGAGTAGTGGTTAACAATTCCGGGGAGCCATGAGTACTAAGATGGGCTCAGTTGAACGTCAACGGGTGCTGCGAGTTCGGCTGACTTCGCACGAATGTTGATCAACGTACCGATTTCTAGCAAAGCTTGGTGCAAGATGAGTGACACGTTCTGAGGAGGAGGTT
>American-crow
GTTATGTAGTAGTTGATCTGTGCGTGTGAAGTAATCCAAACCGTGACTAACAATTCTGGGAGGCCATGAGTACTAAGATAGGCTCAATTGGACGTCAACAGCTGATGTGAGTTCGGCTGACTTTGCAGGAATGTTGTCTAATGTAGCGATTTCGCGTAAAGCATGGTTCAACATGAGCGTCACTTCCTGAAAGGGAGATT
>Emperor-penguin
CTTATGTAGAGGTTGATCCGCACGTGTGCAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAGAATGTGCTCAATTGAACGTCAACGCCTGATGCGAGTTCGGCTGATTTCGCATGAATGTTGCTTAATGTACCGATTTCTAGCAAAGCATGGTGCAACATGAGTGACACGTTCTGAAGGGGAGGTT
>Northern-fulmar
CTTATGTAGAGGTTGATCTGCAAGTGTGAAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAAGATGGGCTCAATTGAACATCAACGGCTGGTGCGAGTTCGGCTGACTTCGCATGAATGTTGATTAATGTACCGATTTCTAGCAAAGCATGATGCCACATGAGTGACACTTTCTGAAGGGGAGGTT
>Carmine-bee-eater
CTTATGTAGAAGTTGTTCTGCACGTGTGAAGTAATCTAAGTAGTGGCTAAAAGTTCTTAGAGGCCATGAGTACTAAGGTGGGCTCAATTGAACGTTAACGGCTGATGCGAGTTCGGCTGATTTCGCATGAATGTCGATTAATGTCCCAATTCCTAGAAAAGCATGGTGCAACATGAGTGACACTTTTTATAGGGGAGGTT
>White-tailed-tropicbird
CTTATGTAGAGGTAGATCAGCACGTGTGAAGTAATGTAAGTAGTAGCTAACAGTTCTGGGAAGCCATGAGCACTAAGACGGGATCAGTCAAACGTCAACGGCTGATGCTGGTTCGGATGACTTCGCATGAATTTTGATTAATGTGCCGATTCCTAGCAAAGCGTGATGCAACATGAGTGACACTGTCTGAAGGGGAGGTT
>Great-cormorant
CTTATGCAGAGGTTGATCTGCACGTGTGAAGTAATGTAAGTAGTGGCTAACACTTCTGGGGAGCCATGAGTACTAAGATGGGCTCAATTGAACGTCAACAGCTGATCCAAGTTCGGCTGACTTCGCATGAGTGTTGATTAATGTTCCGACTTCTAGCAAAGCATATTGCAACATGAGTGACACTTTCTCAAGAGGAGGAT
>Bar-tailed-trogon
CGTACGTAGAAGTTGAGCTGCACGTGTGAAGTCATCTAAGTAGTGGCTAACAGTTCTGGGAGGCCATGAGTATTAAAATGAGCTCGACTAAACGTCAACGGCTGATACGAGTTCGGCTGACTTCGCATAATTGTCGATTAATGCACCGATTTCTAGAAAATTATGGTGCGACATGAGTGACACTTTTAGAAGATGAGGTT
>Chimney-swift
TTTATGTGGGGGTTGATCAGCACGTATGAAGTAATGTAGGTAGTGGCTAACAGTTCTGGGAAGCCATAAGTATTAAGATAGGACCAATTGAACGTCAACGGCTAATGCGAGTTCCGCTGACTTCGCATGGATGTCGATTAGTAAACCGATTTTTAGCAAAGCATGGTGCAACATGAATGATACTTTCTGAGGAGGAGGTC
>Brown-mesite
CTTATGGAAAGGTTGATCTCCACGTGTGAAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGATTACTAAGATAGTCTCAGTTGGACGTCAGCGGCTGATGCGAGCTCGGCCGACTTAGCATGAATGTTGATCAGTGTACCGATTTCTAGCAAAGCATGGTGCAACATGAGTAACACTCTGTGAAAGGGAGGTT
>Medium-ground-finch
GTTATGTAGTAGTTGATATATACGTGTGAAGTAATCCAAATCGTGGCTAACAATTCTGGAAGGCCATAAGTACTAAGATAGGCTCAATTGGACGTCAACACCTGATGAGAGTTCGGCTGACTTTGGAAGAATGTTGTCTAATGTAGCGATTTTGAGGAAAGCATGGTTCAATATGAGCGACACTTCCGGAAATGGAGATT
>White-tailed-eagle
CTTGTGTAAAAATTAATCTGCACGTGTGAAGTAACCTAAGTAGTGGCTAACAGTTCTGGGAGGCCATGAGTACTAAGATGGGCTCAATTGAAAGTCAACCGCTGATGCGAGTTCGGCTGACTTCGAATGAATGTTGTTTAATGTGTCGATTTCTAGCAAAGCACGGTGCAACATAAGTGACACTTTCTGAAGGGGAGGTT
>Crested-ibis
CTTATGTAGAGATTGATCTGCACGTGTGAAGTAATGTAAGTGGTGGCTAACGGCTCTGAGAAGCTATGGATACTAAGATGCGCTCAATTGAACGTCAACGGCTGATGCGAGTTCGGCTGACCTTGCATGAATGTTGATTAATGTACCGATTTCTAGCAAAGCATGGTGCAACATGAGTGACGCTTTCTGAAGGGGAGGTT
>Rifleman
CTCATGTATAAATTGATCTGAACGTGGGAAGTAATCTAAGTCGTGGCTACCAGTTCTGGGAGTCCATGAGTACTAAGGTGGGCTCAATTGGACGTCAACAGCTGAAGTGAGTTTGGCAGACTTCACATGAATGTTGTCTAATGTACCGATTTCTAGCAAAGCATGGTGCAACATGAGCGACACGTCCTGAAAGGGAGGTT
>Turkey-vulture
CTTATGTAGAAGTTGATCTGCACGTGTGGAGTAATCTAAGTAGTGGCTAACAGTTCTGGGAGTCCATGAGTACTAAGATGGGCTCAATTGAACGTCAACGGCTGACGCGAGTTCGGCTGACTTCGCATGAATGTTGATTAATGTACCGATTTCTAGCAAAGCGCGGTGCAACATGAGTGACACTTTCTGAAGGGGAGGTT
>Golden-collared-manakin
CTTATGTAGTAGTTGATCTGTACGTGTGAAGTAACCCGAGTCGTGGTTAACAGTTCCGGGAGGCCATGAATACTAAGATGGGTTCAATTAGACGTCAACAGCTGATGTGAGTTCGGCTGACTTCGCATGAAAGTTGTCTAATGTAGTGACCTCTTGTAAAGTATGGTGCAACATGAGCCACACTTCCTTAAAGAGAGGTT
>American-flamingo
CTTATGTAGAGGATGATCTGCCCGTGTGAAGTAATGTAAGTAGTGGCTAACAATTCTGGGAAGCCATGAGTACTAAGATGGGCTCAAGTGAACGTTAGCGGCTGATGCGAGTTCGGCTGACTTCGCATGAATGTTGATTAATGTACCGATCTCTAGCAAAGTATGGTGCAACATAAGTGACACTTTCTGAAGGGGAGGTT
>Budgerigar
TCGATGTAGCAGTTGATCTACACGTGTGAATTAGTCTAAGTAGTGGCTAACAGTTTTGGGAGGCCATGAGTAGAAAGATGCGCTCAATTGAACGTCAACGGCTGATGCGAGTTCGGCCGACTTCGCTTGAATGTTGATTGATGTGCCGATTCCTAGCGAAGTATGGTGCAACATCAGTGACACTTTCTTAACGGGAGGTT
>Zebra-finch
GTTATGTAGTAGTTGATCTGTACGTGTGAAATAATCCAAATCGTGGCTAACAATTCTGGAAGGCCATAAGTACTAACATAGGCTCAATTGGACGTCAACACCTGATGTGAGTTCGGCGGACTTTGCAAGAATGTTGTCTAATGTATCGATTTCGAGGAAAGCATGGTTCAATATGGGCGACACCTCCTGAAAGGGAAATT
>Grey-crowned-crane
CTTTTGTAGGGGTTGATCTGCACGTGTGAAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAAGATGGGTTCAATTGAACGTCCACGGCTGATGCGAGTTCGGCTGACTCCGCATGAACGTTGATTAATCTACCGATTTCCAGCAAAGCATGGTGCAACATGAGTGACACTTTCTGAAGGGGAGGTT
>Peregrine-falcon
CTTATGTTGAAGTTGATCTGCAGGTGGGAGGTAATCTAAGTAGTGGCTAACAGTTCTGGGAGGCCATGAGTACTAAGATGGGCTCAATTGAACGTCAACGGCTGATGCGAGTTCGGCTGAGTTCGCATACATGTTGATCAATGTAACGATTTCTAGCAAAGCATGGTGCAACATGAGTGACACTGCCTGGAGGGTAGGTT
>Annas-hummingbird
CTTGTGTTGGGGCTGCTCGGAGCGTGTTAAGTAATGTAAGTAGTGGCTAATAGTTCTGGGTAGCCATGAGTACTAAGATAGGCTTGATTGAACGTCAATGGCTGATGCGAGTTCCGTTCACCTCGCATGGATGTTCATTGATGTACCGATTTCTAGCAAAGCGTGGTTGAGCATGAGTGACACTATCGGGGGAGAAGGTC
>Common-ostrich
CTTATGTGTAAGTTGATCGACACGTGTGAAACAATGTAAGTTGTGGCTAACAGTTTTGGGAAGCCATGAGCATTAAGACGGGCTCAATCACCTGTCAACGACTGATATGAGTTCGGCTGACTTCGCGTGGAAGTTGACTAATGTACCGATTTCTTGCATACCATTGTGCATCATGAGCAACGCTTTGTGAAGGATAGCTT
>Speckled-mousebird
GTTATTTAAAAGTTAATCTGCACGTCTGAAGTAATCGAAGTAGTGGCTAACAGTCCTGGGAGGCCGTGGGTACTAAGATGGGTTCCATTAAACGTCAACGGCTGATGCGAGTTCGGCTGGCATCGCATGAATGCTGATTAATGTACCACTTTCTAGCAGAGCATGGTGCAACATGAGTGAGATTTTTTGAAGGGGCGGTT
>Cuckoo-roller
CCTATGTAGAAGTTGATCTCCACGTGGGAAATAATCTAAGTAGTGGCTAACAGTTTTGGGAGGCCCTGAGTACTAAGATGGGCTCAATTGAACGTCAACGGCTGGTGCGAGTTCGGGTGTACTCGCACGAATGTTGATTAATGCACCAATTTCTAGAAAAGCATGGTGCAACATGAGTGACACTTTCTGAAGGGCAAGTT
>Kea
TCGATGTAGCAGTTGATCTGCACGTGTGAATTAATCTAAGTAGTTGATAACAGTTTTAGGAGGCCATGCGTAGAAAGATGCACACAATTGAACGTCAACGCTTGATGCGAGTCCGGCTGACTTCACTTTAGTGTTGATTAATGTGCCGATTTCTAGCAAAGTATGGTGCAACACGAGTGACACTTTATTAACGGGAGGTT
>Dalmatian-pelican
CTTATGTAGAGGTTGATCTGCACGTGTGAAGTAAAGTAAGTAGTGACTAACAGTTCTGGGAAGCCATGAGTACTAAGATGGGTTCAATTGAACGTCAACGGCTGATGCGAGTTCGGCTGACTTCGCATCAATGTCGATTAATATACCGATTTCTAGAAAAGCATGGTGCAACATGAGTTACACTTTCTGATGGTGATGTT
>Yellow-throated-sandgrouse
CTTATGTAGAGGTTGATCTCCATGTGTGGAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGCCATGAGTACTAAGGTGGGCTCGATTGGACGTCAGTGGCTGATGCGAGTTCGACTGACTTCGCATGAATATTGATTAATGTACCGATTTCTAGCAAAGCATGGTGCAACAAGAGGAACACTTCCTGAAAGAAGGGCT
>Bald-eagle
CTTGTGTAAAAATTAATCTGCACGTGTGAAGTAACCTAAGTAGTGGCTAACAGTTCTGGGAGGCCATGAGTACTAAGATGGGCTCAATTGAAAGTCAACCGCTGATGCGAGTTCGGCTGACTTCGAATGAATGTTGTTTAATGTGTCGATTTCTAGCAAAGCACGGTGCAACATAAGTGACACTTTCTGAAGGGGAGGTT
>Little-egret
CTTGTGTAGAGGTTGATCCGCACGTGTGAAGTAATGTAAGTAGTGGCTAACAGTTCTGGGAAGTCATGAATACTAAGATAGGTTCAATTAAACGTCAACGGCTGATACGAGTTCGGCTGACTTCGCATGAATGTTGATTAATGTACCGATTTCCAGCAAAGCATGTTGCTACATGAGTGACACTTTTTGAAGAGGAGGGT
>Rhinoceros-hornbill
CTTATGTAGGGGTTAACCTGCGCGTGTGAAGTCATCTAAGTAGTGGCAAACAGTTCTGAGGGGCCATGAGCACTAAGATGGGCTCAATTCAACGTCAGCGGCTGATGCGAGTTCGGCGGATTTCGCGTGAGTGTCGATTAATGTACCGACTTCTAGAGAAGCATGGTGCAACATGAGTGGCACTTTCTAAAGGGGAGGTT
>White-throated-tinamou
CTTATGTGGAAGTTGCTCTACACGTATGAAACATCGTAAGTAACGACTAGCGATTTTGGGAAGCCATGAGTACTGAGATTGGCTCAACTGAACGTCAACGGCTGACACGAGTTCGCCTGACTTCGCGTGGAGGTCGACTAATGTACCGACTTCCTGCAAACCATGGTGCATCATGAGTAAGGCTTTGTGAAGGATAGATT
>Pigeon
CTTATGTAGAGGTTAGTGTCCACGTGTGAAGTAATGTAAGCAGTGGCTACTAATTCTGGGAAGCCATGAATACTAAGATGGGCTCAATTGAACGTTAGCGACTGATGTGAGTTCGGCTGACTTCGCATGAATGTTGATTAACGTAGCGATTTGAAGCAATGTATGGCGCAATATGAGTAACACTTTCTGAAGGGGAGGTT
