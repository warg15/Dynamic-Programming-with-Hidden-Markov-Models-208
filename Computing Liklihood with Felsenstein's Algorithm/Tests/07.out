-3700.8644638552023
